dev/lint-python

```diff
@@ -71,11 +82,8 @@ function pycodestyle_test {
     # check for locally installed pycodestyle & version
     RUN_LOCAL_PYCODESTYLE="False"
     if hash "$PYCODESTYLE_BUILD" 2> /dev/null; then
-        VERSION=$( $PYCODESTYLE_BUILD --version 2> /dev/null)
-        EXPECTED_PYCODESTYLE=$( ("$PYTHON_EXECUTABLE" -c 'from distutils.version import LooseVersion;
```

Comment:
I think it was mistakenly re-indented at https://github.com/apache/spark/commit/42c48387c047d96154bcfeb95fcb816a43e60d7c#diff-ccd847a0316575dde31bd89786bbe1f2. cc @shaneknapp FYI